docs/data/toolpad/concepts/deployment.md

```diff
@@ -72,6 +72,21 @@ If you want to listen on a specific port you can change the start script to:
   "start": "toolpad start -p 1234"
 ```
 
+## Custom base path
+
+Toolpad applications can run under a custom base path. Use the `--base` CLI parameter to set a base under which the Toolpad application is hosted.
+
+```bash
+toolpad dev --base /foo
+```
+
+Now the Toolpad application is accessible under `http://localhost:3000/foo`. The `--base` parameter must be supplied to the `build` command. A build always has one specific base path:
+
+```bash
+toolpad build --base /foo
```

Comment:
Would it be useful for the slash in the beginning not to be required?

Reply:
> Would it be useful for the slash in the beginning not to be required?

That would take away the option for us to do something useful with paths that don't have a slash. e.g. absolute paths. vite and next.js both expect a leading slash in their base path settings.

Reply:
Ok got it, I wasn't familiar with how other tools do it, this is good then!